client/algod/wrappers.go

```diff
@@ -151,6 +153,34 @@ func (client Client) Block(round uint64, headers ...*Header) (response models.Bl
 	return
 }
 
+func responseReadAll(resp *http.Response, maxContentLength int64) (body []byte, err error) {
+	if resp.ContentLength > 0 {
+		// more efficient path if we know the ContentLength
+		if maxContentLength > 0 && resp.ContentLength > maxContentLength {
+			return nil, errors.New("Content too long")
+		}
+		body = make([]byte, resp.ContentLength)
+		_, err = io.ReadFull(resp.Body, body)
+		return
+	}
+
+	return ioutil.ReadAll(resp.Body)
+}
+
+// BlockRaw gets the raw block msgpack bytes for the given round
+func (client Client) BlockRaw(round uint64, headers ...*Header) (blockbytes []byte, err error) {
+	var resp *http.Response
+	request := struct {
+		Raw string `url:"raw"`
+	}{Raw: "1"}
+	resp, err = client.submitFormRaw(fmt.Sprintf("/block/%d", round), request, "GET", false, headers)
+	if err != nil {
+		return
+	}
+	defer resp.Body.Close()
+	return responseReadAll(resp, 10000000)
```

Comment:
Would naming `10000000` with a constant be more readable here, or would that be wasted breath?